contrib/ceph-reboot-check/update-operator/update-agent.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
```

Comment:
I'm against adding the update-agent and update-operator manifests to Rook repo.
A link to the container linux update operator repo should suffice to not duplicate code.

Reply:
I'm always in a way 'afraid' that another repo will get deleted over time and the files will be lost...
Sounds stupid may be.